package.json

```diff
@@ -50,6 +50,7 @@
     "ngx-bootstrap": "^6.1.0",
     "ngx-cookie-service": "^2.0.2",
     "ngx-disqus": "^2.4.0",
+    "ngx-facebook": "^3.0.0-0",
```

Comment:
As we previously spoke, we don't need the entire facebook library for angular just to have a link to share. 
When adding a new library, we need to wisely choose whether we reeeeally need this library or not. Our application footprint (bundle size) is currently around 12MB.

Very simple solution:

in `component.ts`:
```typescript
    currentLocation = encodeURIComponent(window.location.href);
```

In `.component.html`, for Facebook and Twitter:

```angular
<a href="https://www.facebook.com/sharer/sharer.php?u={{ currentLocation }}"
    target="_blank">
    <i class="fab fa-facebook-f"></i>
</a>

<a href="https://twitter.com/intent/tweet?url={{ currentLocation }}"
    target="_blank">
    <i class="fab fa-twitter"></i>
</a>
```